db/index.js

```diff
@@ -1,27 +1,61 @@
-const db = require('./db.json')
+const { pool } = require('./config')
+const jsonDB = require('./db.json')
+
 const { cleanSIN } = require('../utils')
 
+let runWithJson = false
+
+if (process.env.NODE_ENV === 'test') {
+  runWithJson = true
+} else {
+  pool.query('SELECT NOW()', (err, res) => {
+    if (
+      process.env.NODE_ENV !== 'production' && 
+      (err && err.errno === -61) && 
+      !res
+    ) {
+      pool.end()
+      runWithJson = true
+    }
+  })
+}
```

Comment:
Ah, I see, this is a pretty complex problem. There's going to be some weird ordering about when this callback function is allowed to set the value of `runWithJson` vs when the value is used below.